src/mutations/user/resetWallet.ts

```diff
@@ -0,0 +1,26 @@
+import { ForbiddenError } from 'common/errors'
+import { fromGlobalId } from 'common/utils'
+import { MutationToResetWalletResolver } from 'definitions'
+
+const resolver: MutationToResetWalletResolver = async (
+  root,
+  { input: { id } },
+  { viewer, dataSources: { atomService, userService } }
+) => {
+  const { id: dbId } = fromGlobalId(id)
+  const user = await userService.dataloader.load(dbId)
```

Comment:
does it check current `viewer` user is admin? to make sure only admin can call this API via OSS